### PR TITLE
Installing Samba add-on: rephrase to use a more procedural style

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/common-tasks/file_access.md [hunk 1]
```diff
@@ -19,19 +19,31 @@ Using any of the add-ons listed below, the following directories are made availa
 
 The Samba add-on creates smb shares which can be accessed from another computer. You can also edit files using the editor of your preference from your client computer. This add-on can be installed from the add-on store's official repository.
 
-To configure the Samba add-on, you only need to set a user and password of your choice on the configuration page, save, and then start the add-on. The add-on will not start without setting a password!
+To install the add-on, follow these steps:
 
-To connect to the Samba server from another device, you will use the IP address or hostname of your server. Either of these can be found on the {% my network title="Settings > System > Network" %} page of your UI.
+1. Go to {% my supervisor_addon title="**Settings** > **Add-ons** > **Samba share**" addon="core_samba" %} and select **Install**.
+2. On the **Configuration** tab, define **Username** and **Password**, store them in a save place, and save your changes.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
2. On the **Configuration** tab, define **Username** and **Password**, store them in a safe place, and save your changes.
```

File: source/_includes/common-tasks/file_access.md [hunk 2]
```diff
@@ -19,19 +19,31 @@ Using any of the add-ons listed below, the following directories are made availa
...
+4. To start the add-on, on the **Information** tab, select **Start**.
 
-For connecting from Windows 10, you can enter the IP address or hostname in File Explorer's address bar with two backslashes, as shown in the example screenshot.
+To connect to the Samba server from the other device, follow these steps:
 
-<img src='/images/hassio/screenshots/file_explorer.png' />
+1. Go to {% my network title="**Settings** > **System** > **Network**" %} and take note of the **Host name**.
+   - Alternatively, you can look up the host name or IP address of your Home Assistant on your router.
+2. How you connect from another device to Home Assistant depends on your system. Use one of the following options:
+   - **On Windows**: Open **File Explorer** and in the address bar, enter the IP address or hostname with two backslashes.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]: Maybe a written example here?

File: source/_includes/common-tasks/file_access.md [hunk 3]
```diff
@@ -19,19 +19,31 @@ Using any of the add-ons listed below, the following directories are made availa
...
 
-You should then be prompted for the credentials you entered in the Samba add-on configuration. You also have the option of having the credentials stored so that you do not need to enter them again. After that, you'll have access to the directories which you can then mount as a drive or pin to Quick Access.
+     <img src='/images/hassio/screenshots/file_explorer.png' />
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md]:
```suggestion
     <p class='img'>
         <img src='/images/hassio/screenshots/file_explorer.png' alt='Screenshot of File Explorer displaying the navigation to a file share using an IP address'>
         Screenshot of File Explorer displaying the navigation to a file share using an IP address
     </p>
```

File: source/_includes/common-tasks/file_access.md [hunk 4]
```diff
@@ -19,19 +19,34 @@ Using any of the add-ons listed below, the following directories are made availa
 
 The Samba add-on creates smb shares which can be accessed from another computer. You can also edit files using the editor of your preference from your client computer. This add-on can be installed from the add-on store's official repository.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md:20]: > Samba add-on creates smb shares

Cool! what are those?

I think "smb shares" should be a term here 😄

Comment (ludeeus) [reply source/_includes/common-tasks/file_access.md:20]: as it's an acronym, should probably also be "SMB"

File: source/_includes/common-tasks/file_access.md [hunk 5]
```diff
@@ -19,19 +19,34 @@ Using any of the add-ons listed below, the following directories are made availa
...
-
-
-For OS X, connecting to the shares is a matter of using the Finder menu > Go > Connect to Server...
-You would then enter the IP address or hostname of your Home Assistant OS instance as `smb://your.ha.ip.address` or `smb://homeassistant` and enter your credentials when prompted.
+To install the add-on, follow these steps:
+
+1. Go to {% my supervisor_addon title="**Settings** > **Add-ons** > **Samba share**" addon="core_samba" %} and select **Install**.
+2. On the **Configuration** tab, define **Username** and **Password**, store them in a safe place, and save your changes.
+   - You can specify any username and password.
+   - They are not related to the login credentials you use to log in to Home Assistant or to log in to the computer with which you will use Samba share.
```
Comment (ludeeus) [inline source/_includes/common-tasks/file_access.md:27]: samba share or SMB share?

Comment (ludeeus) [reply source/_includes/common-tasks/file_access.md:27]: samba.

**Review by ludeeus — APPROVED** [review]

File: source/_includes/common-tasks/file_access.md (hunk 4 above)
Comment (c0ffeeca7) [reply source/_includes/common-tasks/file_access.md:20]: Excellent point. Thank you for calling me out on it. Will address this in a separate PR.

Comment (c0ffeeca7) [reply source/_includes/common-tasks/file_access.md:20]: #32913 
removed the mention of the protocol. I think it should be possible to use the add-on without knowing about the underlying protocol.

**Pull request merged**